carmajava/launch/guidance.launch

```diff
@@ -92,8 +92,10 @@
   <!-- Trajectory Executor -->
   <include file="$(find trajectory_executor)/launch/trajectory_executor.launch" />
 
-  <!-- Pure Pursuit Wrapper -->
-  <include file="$(find pure_pursuit_wrapper)/launch/pure_pursuit_wrapper.launch" />
+
+  <rosparam command="load" file="$(arg vehicle_calibration_dir)/mpc_follower/params.yaml"/>
+  <!-- MPC Follower Wrapper -->
+  <include file="$(find mpc_follower_wrapper)/launch/mpc_follower_wrapper.launch" />
```

Comment:
Probably would be better to add and argument to mpc_follower_wrapper.launch that takes in the vehicle_calibration_dir. Then you can move the <rosparam> line inside that launch file. Keeps the launch file self consistent.